examples/offline_inference/vision_language.py

```diff
@@ -1436,6 +1436,29 @@ def run_qwen2_5_omni(questions: list[str], modality: str):
     )
 
 
+# R-4B
+def run_r_vl(questions: list[str], modality: str) -> ModelRequestData:
+    assert modality == "image"
+    model_name = "YannQi/R-4B"
+
+    prompts = [
+            f"<|im_start|>user <image>\n{question}<|im_end|> \
+        <|im_start|>assistant\n"
```

Comment:
```suggestion
        f"<|im_start|>user <image>\n{question}<|im_end|><|im_start|>assistant\n"  # noqa: E501
```

Using newline like that would add extra whitespace to the string